frontend/src/services/ErrorHandler.ts

```diff
@@ -0,0 +1,36 @@
+import {Dispatch} from 'redux'
+import { toast } from 'react-toastify'
+
+import {errorMessage } from '../constants/errorAndSuccessMessages'
+import {userPreferencesRequestFailed} from '../actions/prefAndProfileActions'
+
+export interface IError {
+        field: string
```

Comment:
Is this `field` mean form field? What if there will be errors in multiple fields? What if error will be not form-related?